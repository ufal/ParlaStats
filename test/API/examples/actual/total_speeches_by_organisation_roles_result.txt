[{'organisation.role': 'interparliamentaryFriendshipGroup', 'COUNT(speech.id) as total_speeches': 17013}, {'organisation.role': 'committee', 'COUNT(speech.id) as total_speeches': 9380}, {'organisation.role': 'parliament', 'COUNT(speech.id) as total_speeches': 6220}, {'organisation.role': 'parliamentaryGroup', 'COUNT(speech.id) as total_speeches': 5299}, {'organisation.role': 'subcommittee', 'COUNT(speech.id) as total_speeches': 4803}, {'organisation.role': 'politicalParty', 'COUNT(speech.id) as total_speeches': 2586}, {'organisation.role': 'commission', 'COUNT(speech.id) as total_speeches': 1890}, {'organisation.role': 'delegation', 'COUNT(speech.id) as total_speeches': 1452}, {'organisation.role': 'government', 'COUNT(speech.id) as total_speeches': 383}, {'organisation.role': 'supervisoryBoard', 'COUNT(speech.id) as total_speeches': 201}, {'organisation.role': 'boardOfDirectors', 'COUNT(speech.id) as total_speeches': 14}, {'organisation.role': 'republic', 'COUNT(speech.id) as total_speeches': 4}]

Running time 35.09228515625 ms

+-----------------------------------+------------------------------------+
|         organisation.role         | COUNT(speech.id) as total_speeches |
+-----------------------------------+------------------------------------+
| interparliamentaryFriendshipGroup |               17013                |
|             committee             |                9380                |
|             parliament            |                6220                |
|         parliamentaryGroup        |                5299                |
|            subcommittee           |                4803                |
|           politicalParty          |                2586                |
|             commission            |                1890                |
|             delegation            |                1452                |
|             government            |                383                 |
|          supervisoryBoard         |                201                 |
|          boardOfDirectors         |                 14                 |
|              republic             |                 4                  |
+-----------------------------------+------------------------------------+
