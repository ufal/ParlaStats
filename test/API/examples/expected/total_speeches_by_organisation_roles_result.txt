[{'organisation.role': 'interparliamentaryFriendshipGroup', 'COUNT(speech.id) as total_speeches': 982430}, {'organisation.role': 'committee', 'COUNT(speech.id) as total_speeches': 402190}, {'organisation.role': 'subcommittee', 'COUNT(speech.id) as total_speeches': 305743}, {'organisation.role': 'parliament', 'COUNT(speech.id) as total_speeches': 253301}, {'organisation.role': 'parliamentaryGroup', 'COUNT(speech.id) as total_speeches': 191839}, {'organisation.role': 'politicalParty', 'COUNT(speech.id) as total_speeches': 123225}, {'organisation.role': 'commission', 'COUNT(speech.id) as total_speeches': 121414}, {'organisation.role': 'delegation', 'COUNT(speech.id) as total_speeches': 28272}, {'organisation.role': 'government', 'COUNT(speech.id) as total_speeches': 23025}, {'organisation.role': 'supervisoryBoard', 'COUNT(speech.id) as total_speeches': 8141}, {'organisation.role': 'boardOfDirectors', 'COUNT(speech.id) as total_speeches': 3477}, {'organisation.role': 'institution', 'COUNT(speech.id) as total_speeches': 71}, {'organisation.role': 'republic', 'COUNT(speech.id) as total_speeches': 24}]

+-----------------------------------+------------------------------------+
|         organisation.role         | COUNT(speech.id) as total_speeches |
+-----------------------------------+------------------------------------+
| interparliamentaryFriendshipGroup |               982430               |
|             committee             |               402190               |
|            subcommittee           |               305743               |
|             parliament            |               253301               |
|         parliamentaryGroup        |               191839               |
|           politicalParty          |               123225               |
|             commission            |               121414               |
|             delegation            |               28272                |
|             government            |               23025                |
|          supervisoryBoard         |                8141                |
|          boardOfDirectors         |                3477                |
|            institution            |                 71                 |
|              republic             |                 24                 |
+-----------------------------------+------------------------------------+
